hack/remove-calico-policy/iptables-remover-ds.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: calico-remove-iptables
+  namespace: kube-system
+  labels:
+    k8s-app: calico-remove-iptables
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: calico-remove-iptables
+    spec:
+      hostNetwork: true
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                - key: projectcalico.org/ds-ready
+                  operator: NotIn
+                  values:
+                  - "true"
+      tolerations:
+        - key: CriticalAddonsOnly
+          operator: Exists
+      containers:
+        - name: calico-remove-iptables
+          image: calico/iptables-remover
```

Comment:
so im wondering, if you have to build this image, it would only be built on one node, and if this DaemonSet is spun up on another node it will try to look for that image and if it's not there it will try to pull it from dockerhub and will fail, right?